### ISSUE TITLE
print/println does not accept null.

### ISSUE THREAD

**Issue by y-uti** [php-java/php-java#199]
# Tell me your versions
- **PHP**
```
$ php -v
PHP 7.3.2 (cli) (built: Feb 17 2019 17:13:31) ( NTS )
Copyright (c) 1997-2018 The PHP Group
Zend Engine v3.3.2, Copyright (c) 1998-2018 Zend Technologies
    with Zend OPcache v7.3.2, Copyright (c) 1999-2018, by Zend Technologies
```
- **JDK**
```
$ java -version
openjdk version "1.8.0_212"
OpenJDK Runtime Environment (build 1.8.0_212-b04)
OpenJDK 64-Bit Server VM (build 25.212-b04, mixed mode)
$ javac -version
javac 1.8.0_212
```
- **PHPJava**
```
$ git log -n 1 --oneline
64f9d24 Merge pull request #196 from php-java/enhance-invokeinterface
```
# Summary

System.out.println (and also print) method does not accept null.

# Code

```
public class PrintlnTest {
    public static void main(String[] args) {
        String s = null;
        System.out.println(s);
        char[] c = null;
        System.out.println(c);
    }
}
```

# Expect

```
$ java PrintlnTest
null
Exception in thread "main" java.lang.NullPointerException
        at java.io.Writer.write(Writer.java:127)
        at java.io.PrintStream.write(PrintStream.java:503)
        at java.io.PrintStream.print(PrintStream.java:653)
        at java.io.PrintStream.println(PrintStream.java:792)
        at PrintlnTest.main(PrintlnTest.java:6)
```

# Actual

(PrintlnTest.php simply invokes main method of PrintlnTest.class)

```
$ php PrintlnTest.php
PHP Fatal error:  Uncaught PHPJava\Packages\java\util\IllegalFormatException: Cannot pass "NULL" in PHPJava\Packages\java\io\PrintStream::println in /.../vendor/php-java/php-java/src/Packages/java/io/PrintStream.php:34
Stack trace:
#0 [internal function]: PHPJava\Packages\java\io\PrintStream->println(NULL)
#1 /.../vendor/php-java/php-java/src/Core/JVM/Invoker/Extended/PHPMethodCallable.php(47): ReflectionMethod->invokeArgs(Object(PHPJava\Packages\java\io\PrintStream), Array)
#2 /.../vendor/php-java/php-java/src/Kernel/Mnemonics/_invokevirtual.php(72): PHPJava\Core\JVM\Invoker\PHPClassMethodInvoker->call('println', NULL)
#3 /.../vendor/php-java/php-java/src/Core/JVM/Invoker/Extended/JavaMethodCallable.php(264): PHPJava\Kernel\Mnemonics\_invokevirtual->execute()
#4 /.../examples/PrintlnTest/PrintlnTest.php(15): PHPJava\Core\JVM\Invo in /.../vendor/php-java/php-java/src/Kernel/Core/ExceptionTableInspectable.php on line 67
```

**Comment by m3m0r7** [comment]
@y-uti 
It seems that `print` and `println` does not haves processing conditions for null.
I added conditions. No problem?

https://github.com/php-java/php-java/pull/207

**Comment by y-uti** [comment]
#207 doesn't solve this issue.
I'm sorry for the unclearness, but the expected behavior is:

output "null" if the argument is **String** null
```
String s = null;
System.out.println(s); // outputs "null"
```

in contrast, throw exception if the argument is **char[]** null.
```
char[] c = null;
System.out.println(c); // throws exception
```

(Now I think this may be difficult for PHPJava, because it seems that PHPJava doesn't distinguish the type of null values.)

**Comment by m3m0r7** [comment]
@y-uti 
Thanks! Okay, PHPJava can solves it.
Because PHPJava has parameter descriptor information. (a.k.a `L;I;java.lang.String` value)
I'm thinking that I can solve the problem by using it.

**Comment by m3m0r7** [comment]
Sorry for late update.

Fixed it: #209

I added new annotation `@depended-info`.
It is providing method parameters type, method return type, method info from constant pool and so on.

By this updating, you can pass provided any parameter to Java Package, And it is able to flexible method processing by above annotations.

**Comment by y-uti** [comment]
Hi, print/println don't throw an exception for null string case in Java. I fixed it in #224.

**Comment by m3m0r7** [comment]
Thanks!
